terraform/deploy/peering_concourse.tf

```diff
@@ -4,4 +4,13 @@ resource "aws_route" "route" {
   destination_cidr_block    = local.cidr_block[local.environment].mon-master-vpc
   vpc_peering_connection_id = module.vpc.outputs.prometheus_pcx
 }
-  
+
+resource "aws_security_group_rule" "web_lb_in_metrics" {
+  description       = "inbound traffic to web nodes metrics port"
+  from_port         = 9090
+  protocol          = "tcp"
+  security_group_id = data.terraform_remote_state.aws_concourse.outputs.concourse_web_sg
+  to_port           = 9090
+  type              = "ingress"
+  cidr_blocks       = [local.cidr_block[local.environment].mon-master-vpc]
```

Comment:
Can you change this to `source_security_group_id` please, pointing it at the prometheus SG? We generally prefer SG rules to be based on SGs rather than IPs so that they're narrower in range.

Reply:
Resolved